src/components/forms/TransitGatewayForm.js

```diff
@@ -51,29 +55,50 @@ class TransitGatewayForm extends Component {
     this.setState(this.eventTargetToNameAndValue(event));
   }
 
+  /**
+   * Handle crn input
+   * @param {event} event
+   */
+  handleCRNs(event) {
+    invalidRegex;
+    let crnList = event.target.value
+      ? event.target.value
+          .replace(/\s\s+/g, "") // replace extra spaces
+          .replace(/,(?=,)/g, "") // prevent null tags from
+          .replace(/[^\w,-:]/g, "")
+          .split(",")
+      : [];
+    this.setState({ crns: crnList });
+  }
+
+  /**
+   * Invalid crns
+   * @param {Array} crns
+   */
+  // https://cloud.ibm.com/docs/account?topic=account-crn
+  invalidCRNs(crns) {
+    if (crns.length === 0) return false;
+    for (let crn of crns) {
+      if (crn.match(this.props.crnRegex) === null) {
+        return true;
+      }
+    }
+    return false;
+  }
```

Comment:
i think this could go into a utils file and have a ut and we shouldn't need to pass crnRegex as a prop since it should always be the same